### ISSUE TITLE
Training Code for AudioLDM2

### ISSUE THREAD

**Issue by Tortoise17** [haoheliu/AudioLDM-training-finetuning#24]
This is really a great implementation. Is there audioldm2 code and config already available?

**Comment by jjyaoao** [comment]
Same question, very much looking forward to it!

**Comment by SoshyHayami** [comment]
yeah I'm curios about it too. I'll stick with audiocraft unless AudioLDM2 is fully open-sourced. 
it's a pity, this model is vastly superior to anything else I've ever seen, wish We could have some control over fine-tuning the weights.

**Comment by Tortoise17** [comment]
@haoheliu is there any update or help possible on this issue? training for text to speech audioldm2 model.

**Comment by MoayedHajiAli** [comment]
@haoheliu Do you have any updates on this?

**Comment by Tortoise17** [comment]
@haoheliu Is there any hope for this training checkpoints and repo for AudioLDM2? and is there any possibility to fine tune the transcription base audioLDM2 model?

**Comment by Tortoise17** [comment]
@haoheliu Is there any plan to release the checkpoints for the TTS model to finetune? and any possibility to have its opensource license for gigaspeech or liberispeech tts ?
